spec.bs

```diff
@@ -59,16 +60,250 @@ A <dfn>import map</dfn> is a [=struct=] with two [=struct/items=]:
 
 <h2 id="acquiring">Acquiring import maps</h2>
 
-TODO...
+<h3 id="integration-environment-settings-object">New members of environment settings object</h3>
+
+Each [=environment settings object=] will get an <dfn for="environment settings object">import map</dfn> algorithm, which returns an [=/import map=] created by parsing and merging all `<script type="importmap">` elements that are encountered (before the cutoff).
+
+A <a spec="html">Document</a> has an [=/import map=] <dfn for="Document">import map</dfn>.
+It is initially null.
```

Comment:
Shouldn't it be an empty import map? Something like "it is initially an import map with an imports and scopes both being empty maps".

Reply:
Done. Defined "empty import map".